src/styles/components/_checkbox.scss

```diff
@@ -0,0 +1,30 @@
+@import '../settings/colors';
+
+.w-checkbox__label {
+  cursor: pointer;
+  display: inline-flex;
+  position: relative;
+}
+
+.w-checkbox__input {
+  -moz-appearance: none;
+  -webkit-appearance: none;
+  appearance: none;
+  background: url('/images/icons/check_box_outline_blank.svg');
+  border: 2px solid $WHITE;
+  border-radius: 4px;
+  cursor: pointer;
+  height: 28px;
+  margin: 0px 30px 0px 0px;
+  outline: none;
+  transition-duration: .2s;
```

Comment:
Are we using this on our other focus styles? Feels a bit odd to me.

btw we do have a variable for this in `settings/gobal.scss`